blocks/text/text.css

```diff
@@ -36,24 +36,20 @@ main .block.text.black p {
 }
 
 main .block.text.highlight {
-  margin-top: 5rem;
+  margin-top: 1.5rem;
+  margin-bottom: 5rem;
+  padding: 1.875rem 2.5rem;
+  background: var(--transparent-blue-light-color);
+  border-left: 5px solid var(--health-beauty-orange);
 }
 
-
-main .block.text.highlight h2 {
-  margin-bottom: .5rem;
-  text-align: center;
+main .block.text.highlight .text-title {
```

Comment:
I am wondering if we can ask Dejan about the title.
Because i don't think its important to have it in orange color.
If he agrees on this, then we can also remove the JS code.

Reply:
As we talked, will keep it